player/public/js/main.js

```diff
@@ -296,7 +296,18 @@ function setGlobalConfig (configuration) {
 		_.extend(metadata, _.pick(configuration.metadata, "hierarchyInfo", "isAvailableLocally", "basePath", "rollup"))
 		metadata.basepath = metadata.basePath
 		configuration.basepath = configuration.basePath
-		if (metadata.rollup) { configuration.rollup = metadata.rollup }
+
+		if(configuration.contextRollup){
+			configuration.rollup = configuration.contextRollup;
+		}
+		
+		configuration.object = {}
```

Comment:
if anyone sending object your replacing with an empty object. use _.assign to fix this